tutorindigo/templates/indigo/lms/templates/instructor/instructor_dashboard_2/extensions.html

```diff
@@ -0,0 +1,120 @@
+<%page args="section_data" expression_filter="h"/>
```

Comment:
What is the reason for copying this file from upstream? If there were a11y issues in this template, they should have been fixed upstream instead of copying the file. 
@zubairshakoorarbisoft do you have additional context on this?